doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst

```diff
@@ -0,0 +1,1265 @@
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+|no_changes_yet_note|
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``a768a05e6205e415564226543cee67559d15b736``, with some |NCS| specific additions.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into |NCS| since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline a768a05e62 ^4bbd91a908
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^a768a05e62
+
+The current |NCS| main branch is based on revision ``a768a05e62`` of Zephyr.
+
+.. note::
+   For possible breaking changes and changes between the latest Zephyr release and the current Zephyr version, refer to the :ref:`Zephyr release notes <zephyr_release_notes>`.
+
+Additions specific to |NCS|
+---------------------------
+
+|no_changes_yet_note|
+
+zcbor
+=====
+
+|no_changes_yet_note|
+
+Trusted Firmware-M
+==================
+
+* Added a section :ref:`tfm_encrypted_its` describing Internal Trusted Storage (ITS) with encryption.
+
+cJSON
+=====
+
+|no_changes_yet_note|
+
+Documentation
+=============
+
+* Added:
+
+  * :ref:`create_application` page that provides information about the applications available in the |NCS| and how to create them.
+  * A page on :ref:`ug_wireless_coexistence` in :ref:`protocols`.
+  * A page on :ref:`ug_sidewalk` in :ref:`protocols`.
+  * Pages on :ref:`thread_device_types` and :ref:`thread_sed_ssed` to the :ref:`ug_thread` documentation.
+  * A new page :ref:`ug_pmic`, containing :ref:`ug_npm1300_features` and :ref:`ug_npm1300_gs`.
+  * A section about :ref:`nrf70_gs_shields_expansion_boards` in :ref:`nrf7002dk_nrf5340` user guide.
+  * A page on :ref:`ug_nrf70_developing_scan_operation` in the :ref:`ug_nrf70_developing` user guide.
+  * The :ref:`ug_bt_qualification` page in :ref:`protocols`.
+  * A section on Wi-Fi in the :ref:`app_memory` page.
+  * Own page for :ref:`bt_mesh_samples`.
+  * :ref:`migration_2.5`.
+
+* Updated:
+
+  * :ref:`samples` with separate sections for :ref:`keys_samples` and :ref:`peripheral_samples`, which were previously listed in :ref:`other_samples`.
+  * The :ref:`emds_readme` library documentation with :ref:`emds_readme_application_integration` section about the formula used to compute the required storage time at shutdown in a worst case scenario.
+  * The structure of the :ref:`nrf_modem_lib_readme` documentation.
```

Comment:
Apart from structure change, Modem tracing with RTT section was added as part of the PR (https://github.com/nrfconnect/sdk-nrf/pull/11440/files). May be this could be added as a change log entry? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/modem/nrf_modem_lib/nrf_modem_lib_trace.html#modem-tracing-with-rtt

Reply:
Done